tests/verifyPMMInventory_test.js

```diff
@@ -1,7 +1,32 @@
 const assert = require('assert');
+const faker = require('faker');
+
+const pmmManagerCmd = '/srv/pmm-qa/pmm-tests/pmm-framework.sh';
+const mysqlServiceName = `mysql-push-mode-${faker.datatype.number()}`;
+const postgresServiceName = `postgres-push-mode-${faker.datatype.number()}`;
+const mongoServiceName = `mongo-push-mode-${faker.datatype.number()}`;
+
+const services = new DataTable(['serviceName']);
+
+services.add([mysqlServiceName]);
+services.add([postgresServiceName]);
+services.add([mongoServiceName]);
+services.add(['haproxy']);
 
 Feature('Inventory page');
 
+BeforeSuite(async ({ I }) => {
+  I.say(await I.verifyCommand(`${pmmManagerCmd} --addclient=ps,1 --deploy-service-with-name ${mysqlServiceName}`));
+  I.say(await I.verifyCommand(`${pmmManagerCmd} --addclient=pdpgsql,1 --deploy-service-with-name ${postgresServiceName}`));
+  I.say(await I.verifyCommand(`${pmmManagerCmd} --addclient=mo,1 --deploy-service-with-name ${mongoServiceName}`));
```

Comment:
@vyarosh can we use the same approach for addind `haproxy` to avoid `if` statement in scenario?

Reply:
As discussed with Puneet lets just skip it for now. Thanks